LocalWiki.php

```diff
@@ -27,6 +27,36 @@ function onBeforePageDisplay( OutputPage $out ) {
 			$out->addMeta( 'og:image:width', '1200' );
 		}
 
+		break;
+	case 'gratisdatawiki':
+		$wgHooks['OutputPageParserOutput'][] = 'onOutputPageParserOutput';
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
+		
+		function onOutputPageParserOutput( OutputPage $out, ParserOutput $parserOutput ) {
+			$placeholders = $parserOutput->getExtensionData( 'wikibase-view-chunks' );
+			if ( $placeholders !== null ) {
+				$out->setProperty( 'wikibase-view-chunks', $placeholders );
+			}
+			$termsListItems = $parserOutput->getExtensionData( 'wikibase-terms-list-items' );
+			if ( $termsListItems !== null ) {
+				$out->setProperty( 'wikibase-terms-list-items', $termsListItems );
+			}
+			$meta = $parserOutput->getExtensionData( 'wikibase-meta-tags' );
+			$out->setProperty( 'wikibase-meta-tags', $meta );
+			$alternateLinks = $parserOutput->getExtensionData( 'wikibase-alternate-links' );
+			if ( $alternateLinks !== null ) {
+				foreach ( $alternateLinks as $link ) {
+					$out->addLink( $link );
+				}
+			}
+		}
+				
+		function onBeforePageDisplay( OutputPage $out ) {
+			$out->addMeta( 'og:image:width', '1200' );
+			$out->addMeta( 'og:image', wfExpandUrl( $thumb->getUrl(), PROTO_CANONICAL ) );
```

Comment:
This will definitely cause exceptions, since `$thumb` doesn't exist. But this is the default added by PageImages when thumb does exist, so not necessary, since you have that enabled.

Reply:
> This will definitely cause exceptions, since `$thumb` doesn't exist. But this is the default added by PageImages when thumb does exist, so not necessary, since you have that enabled.

that's true